server/sonar-server/src/main/java/org/sonar/server/computation/queue/InternalCeQueueImpl.java

```diff
@@ -142,4 +172,73 @@ public boolean isPeekPaused() {
     return peekPaused.get();
   }
 
+  /**
+   * A {@link PrintWriter} subclass which enforces that line returns are {@code \n} whichever the platform.
+   */
+  private static class LineReturnControllerPrintStream extends PrintWriter {
+
+    LineReturnControllerPrintStream(OutputStream out) {
+      super(out);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this use of "<init>" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1943)